CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+0.10.1 (Apr 20, 2021)
+====================
+
+- [libmamba] add experimental mode that unlock edge features #858
+- [micromamba] add `--experimental` umamba flag to enable experimental mode #858
+- [libmamba] improve base env creation #860
+- [libmamba] fix computation of weakly canonical target prefix #859
+- update libsolv dependency in env-dev.yml file, update documentation (thanks @Aratz) #843
+- [libmamba] handle package cache in secondary locations, fix symlink errors (thanks wenjuno) #856
+- [libmamba] fix cURL SSL error on macos due to Darwin backend remove from conda-forge build (thanks @wolfv) #865
```

Comment:
we can remove that since that was just a CI fix

Reply:
hum yeah but we could also maybe still mention it with less details to keep track of what was done ?
as we also mentioned what we've done to support ccp-filesystem update